### PR TITLE
Remove Suppress annotation from UnusedImports

### PR DESCRIPTION
**Pull request by schalkms**

### CODE REVIEW

File: detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnusedImports.kt [hunk 1]
```diff
@@ -84,13 +84,16 @@ class UnusedImports(config: Config) : Rule(config) {
...
-                .filter { !componentNRegex.matches(it.identifier()!!) }.toList()
+            imports = importList.imports.asSequence()
+                .filter { it.isValidImport }
+                .filter {
+                    val identifier = it.identifier()
+                    identifier?.contains("*")?.not() == true &&
+                            !operatorSet.contains(identifier) &&
+                            !componentNRegex.matches(identifier)
+                }
+                .toList()
```
Comment (BraisGabin) [inline detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnusedImports.kt:96]: Does this improve readability? It removes an unsafe call but I think that it is not worth it

Comment (BraisGabin) [reply detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnusedImports.kt:96]: But, I like to move the `.toList()` and the first `filter` to their own line

Comment (cortinico) [reply detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnusedImports.kt:96]: > Does this improve readability? 

I believe it does as the bigger `filter{}` is all logically related to filtering out identifiers.

**Review by cortinico — APPROVED** [review]

File: detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnusedImports.kt (hunk 1 above)
Comment (schalkms) [reply detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnusedImports.kt:96]: I agree with Nico. Furthermore, we get rid of the **!!**.

**Pull request merged**